setup.py

```diff
@@ -159,7 +159,7 @@ def run(self):
         "dev": dev_requirements,
     },
     url="https://github.com/CSHS-CWRA/ravenpy",
-    version="0.1.6",
+    version="0.1.7",
```

Comment:
We use `bumpversion` to increment the version, which is set to modify 3 or 4 different files (including this one).